global_install_scripts/install.bash

```diff
@@ -408,7 +408,10 @@ END_OF_SCRIPT
 	setup_git_template_talisman_hook
 	echo
 	echo "Setting up talisman hook recursively in git repos"
-	read -p "Please enter root directory to search for git repos (Default: ${HOME}): " SEARCH_ROOT
+	
+	if [[ "$SEARCH_ROOT" == "" ]]; then
```

Comment:
This works if you run it with SEARCH_ROOT as set. 
But when its not set, you would see this error: global_install_scripts/install.bash: line 412: SEARCH_ROOT: unbound variable
I'd recommend declaring it at the top to prevent this, similar to how DEBUG and FORCE_DOWNLOAD are initialised here: https://github.com/thoughtworks/talisman/blob/master/global_install_scripts/install.bash#L6

```
SEARCH_ROOT=${SEARCH_ROOT:-''}

```

Reply:
https://github.com/thoughtworks/talisman/pull/349/commits/6da832ad81df7d2e6c3315acad392debd7e80ed6